GNUmakefile

```diff
@@ -22,6 +22,11 @@ clean:
 release:
 	@git tag ${NEXT_VERSION} && git push --mirror
 	@echo "Pushed ${NEXT_VERSION}"
+	GOPROXY=https://proxy.golang.org GO111MODULE=on go get github.com/jfrog/terraform-provider-artifactory@v${NEXT_VERSION}
+	@echo "Updated pkg cache"
+
+update_pkg_cache:
+	GOPROXY=https://proxy.golang.org GO111MODULE=on go get github.com/jfrog/terraform-provider-artifactory@v${VERSION}
```

Comment:
What's with this go proxy?

Reply:
This is where [https://pkg.go.dev/](https://pkg.go.dev/about#adding-a-package) looks for the package meta data.